lib/evaluation/evaluationClientWrapper.ts

```diff
@@ -427,4 +438,33 @@ export class EvaluationClientWrapper {
     }
     return status;
   }
+
+  private updateSubmissionRequirementMatchPathToAlias(
+    submissionRequirementMatch: SubmissionRequirementMatch,
+    alias: string
+  ) {
+    const matches: string[] = [];
+    submissionRequirementMatch.matches.forEach((m) => {
+      matches.push(m.replace('$', '$.' + alias));
+    });
+    submissionRequirementMatch.matches = matches;
+    if (submissionRequirementMatch.from_nested) {
+      submissionRequirementMatch.from_nested.forEach((f) => {
```

Comment:
Does that make sense? A nested VC is not exposed at the root, but part of another VC right?

Reply:
this comment and the one below made me go through the docs once more and I also had a semi-brief talk with Hafeez about this. the naming of `from_nested` is the problem, you might think that it's talking about the nested credentials or something like this, but in fact it's talking about nested conditions

Reply:
Okay, then the bigger question I guess becomes. Is our current implementation conforming to the `from_nested`. Does it take into account?

Reply:
I don't think so. There are still doubts about the structure of a nested VC.

Reply:
from_nested is not related to nested credentials. and we have several examples for handling them (examples are from [here](https://identity.foundation/presentation-exchange/#submission-requirement) I believe)

Reply:
Yes I get that they are not related to nested credentials now. I was asking whether we have implemented it properly because you had to look it up and Maikel is also not sure

Reply:
The fact that we have examples, doesn't necessarily prove we have a correct solution if the core assumption could have been wrong from the beginning ;)

Reply:
I'll look it up

Reply:
I assume that we can easily find out if a credential is nested based on the filter property from the input descriptor. On the other hand, how do I find out the root of the credential to use in the presentation submission? Can I assume that it is a property called verifiableCredential? Is it an array or object?

Reply:
we're not talking about nested_credentials here. in this PR we have a property called `verifiableCredential: VerifiableCredential[]` to look into

Reply:
It look's like nested credentials all the time. From the nested_from perspective I'm confident about the implementation.